components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -116,6 +119,18 @@ internal inline fun <U> rustCall(syncOn: Any, callback: (RustError.ByReference)
     }
 }
 
+@Suppress("TooGenericExceptionThrown")
```

Comment:
It's unfortunate this is duplicated from push - do you see any scope to share this in a followup?

Reply:
Can't prior to #1103, since places_destroy_string != push_destroy_string. After that we can find a place for shared dtors and such, potentially.